indico/modules/designer/client/js/index.js

```diff
@@ -462,9 +469,13 @@
       width: function() {
         selectedItem.width = Math.round($('.js-element-width').val() * pixelsPerCm);
         if (selectedItem.type === 'ticket_qr_code') {
```

Comment:
shouldn't this use `isImage` as well?

Reply:
No, this is specific to ticket_qr_code because it is squared: so if you modify width, I set the height accordingly

Reply:
OK fair enough